src/sentry/static/sentry/app/constants/index.tsx

```diff
@@ -13,11 +14,13 @@ export const API_SCOPES = [
   'team:write',
   'team:admin',
   'event:read',
+  'event:write',
```

Comment:
Adding typing to this file actually revealed that we were missing some scopes from this list :)